### PR TITLE
Added setting to turn off login experience v2

### PR DESCRIPTION
**Pull request by dcaro**

### CODE REVIEW

File: docs-ref-conceptual/authenticate-azure-cli-interactively.md [hunk 1]
```diff
@@ -79,6 +79,13 @@ Here are some guidelines about the subscription selector to keep in mind:
 * The subscription selector is only available when using the `az login` command.
 * You aren't prompted to select a subscription when you're logging in with a service principal or managed identity.
 
+If you encounter any issue and want to rever to the previous login experience, run the following commands:
+
+```azurecli-interactive
+az config set core.login_experience_v2=off
+az login
+```
+
```
Comment (jiasli) [inline docs-ref-conceptual/authenticate-azure-cli-interactively.md:88]: This was originally documented at https://github.com/Azure/azure-cli/pull/28910.

**Review by jiasli — APPROVED** [review]

File: docs-ref-conceptual/authenticate-azure-cli-interactively.md (hunk 1 above)
Comment (dbradish-microsoft) [reply docs-ref-conceptual/authenticate-azure-cli-interactively.md:88]: This code block is in lines 107 - 109

File: docs-ref-conceptual/authenticate-azure-cli-interactively.md [hunk 2]
```diff
@@ -79,6 +79,13 @@ Here are some guidelines about the subscription selector to keep in mind:
 * The subscription selector is only available when using the `az login` command.
 * You aren't prompted to select a subscription when you're logging in with a service principal or managed identity.
 
+If you encounter any issue and want to rever to the previous login experience, run the following commands:
```
Comment (dbradish-microsoft) [inline docs-ref-conceptual/authenticate-azure-cli-interactively.md:82]:
```suggestion
If want to disable the subscription selector feature, run the following commands:
```

File: docs-ref-conceptual/authenticate-azure-cli-interactively.md [hunk 3]
```diff
@@ -79,6 +79,13 @@ Here are some guidelines about the subscription selector to keep in mind:
...
+
+```azurecli-interactive
+az config set core.login_experience_v2=off
```
Comment (dbradish-microsoft) [inline docs-ref-conceptual/authenticate-azure-cli-interactively.md:85]:
```suggestion
az account clear
az config set core.login_experience_v2=off
```

Comment (dbradish-microsoft) [reply docs-ref-conceptual/authenticate-azure-cli-interactively.md:85]: @jiasli , @dcaro , the `az clear` isn't required, it just seems to make the shut-off of the feature cleaner in my thinking.  If there is a definite reason NOT to include it in the code block, please <resolve> my suggestion w/o adding the line.

**Comment by dbradish-microsoft** [comment]
Merged after discussion with @dcaro

**Pull request merged**